File: src/Components/Posts/Posts.js
//CSS
import './Posts.css'

//Hooks
import { useState, useEffect } from 'react'

//Components
import PostCard from '../PostCard/PostCard'

export default  function Posts({ allPosts }){
    return (
        <div className="container-posts">
            {allPosts.map(post => {
                return (
                    <PostCard
                        key={post.postId}
                        nome={(post.nome)}
                        uf={post.uf}
                        cidade={post.cidade}
                        referencia={post.referencia}
                        descricao={post.descricao}
                        data={post.data}
                    />
                )
            })}
        </div>
    )
}

File: WebAPI-SqlServer/WebAPI/Models/Posts.cs
﻿namespace WebAPI.Models
{
    public class Post
    {
        public int postId { get; set; }
        public string Nome { get; set; }
        public string UF { get; set; }
        public string Cidade { get; set; }
        public string? Referencia { get; set; }
        [StringLength(100)]
        public string Descricao { get; set; }
        public DateTime Data { get; set; }
    }
}
